float64 emotion0  # 高兴   
float64 emotion1  # 信任
float64 emotion2  # 期待
float64 emotion3  # 悲伤  
float64 emotion4  # 愤怒  
float64 emotion5  # 害怕
float64 emotion6  # 厌恶
float64 emotion7  # 无聊  
